extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -1,20 +1,89 @@
 <?php
 
 class ArticleAsJson extends WikiaService {
-	static $media = [];
-	static $users = [];
+	static $media = [ ];
+	static $users = [ ];
 	static $mediaDetailConfig = [
 		'imageMaxWidth' => false
 	];
 
-	const ICON_MAX_SIZE = 48;
 	const CACHE_VERSION = '0.0.3';
+	const CACHE_VERSION_FOR_SEO_FRIENDLY_IMAGES = 1; // for $wgEnableSeoFriendlyImagesForMobile
+
+	const ICON_MAX_SIZE = 48;
+	// Line height in Mercury
+	const ICON_SCALE_TO_MAX_HEIGHT = 20;
+	const MAX_MERCURY_IMAGE_WIDTH = 985;
```

Comment:
On the other hand this describes what this value represents not where it's used.
If it's misleading I can add comment in `getMediaDetailWithSizeFallback` why we use this value, makes sense?